documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorage.java

```diff
@@ -595,7 +595,7 @@ public AbstractNode createFolder(long ownerId,
       }
       //no need to this object later make it eligible to the garbage collactor
       nodeAccessList = null;
-      String name = Text.escapeIllegalJcrChars(cleanName(title.toLowerCase()));
+      String name = Text.escapeIllegalJcrChars(cleanFolderName(title.toLowerCase()));
```

Comment:
Why should we create a new function for folders, let's use the same one and clean both parts for files and folders .

Reply:
done !

Reply:
done!